src/main/java/software/amazon/awssdk/crt/CRT.java

```diff
@@ -103,14 +103,15 @@ private static void loadLibraryFromJar() {
             Path libTempPath = Files.createTempFile(Paths.get(System.getProperty("java.io.tmpdir")), prefix, libraryName);
 
             // open a stream to read the shared lib contents from this JAR
-            InputStream in = CRT.class.getResourceAsStream(libraryPath);
-            if (in == null) {
-                throw new IOException("Unable to open library in jar for AWS CRT: " + libraryPath);
-            }
-            // Copy from jar stream to temp file
-            Files.deleteIfExists(libTempPath);
-            Files.copy(in, libTempPath);
-            in.close();
+            try (InputStream in = CRT.class.getResourceAsStream(libraryPath)) {
+                if (in == null) {
+                    throw new IOException("Unable to open library in jar for AWS CRT: " + libraryPath);
+                }
+                // Copy from jar stream to temp file
+                Files.deleteIfExists(libTempPath);
+                Files.copy(in, libTempPath);
+                in.close();
```

Comment:
`in.close()` will already be called due to the try with resources call won't it?

Reply:
You are correct, fixed!